src/component.js

```diff
@@ -131,7 +131,9 @@ function renderComponent(component) {
 			parentDom.ownerSVGElement !== undefined,
 			null,
 			commitQueue,
-			oldDom == null ? getDomSibling(vnode) : oldDom
+			oldDom == null ? getDomSibling(vnode) : oldDom,
+			null,
```

Comment:
```suggestion
			undefined,
```

Is how it was before

Reply:
This is the `isHydrating` argument so perhaps `false` would be better? Minifies to `!1` instead `void 0` I think